testsuite/features/support/cobbler_test.rb

```diff
@@ -44,6 +59,14 @@ def get_list(what)
     result
   end
 
+  ##
+  # It creates a new distribution, sets the name, kernel, initrd, and breed, and then saves the distribution
+  #
+  # Args:
+  #   name: The name of the distribution.
+  #   kernel: The path to the kernel file.
+  #   initrd: The initrd file for the distribution.
+  #   breed: The type of distribution.  This can be one of the following:. Defaults to suse
```

Comment:
@nodeg do you want to add the possible values here? I saw you were the one who did this part and I'm not being able to find them :sweat_smile:

Reply:
I will have a look. Otherwise I will ask Enno.

Reply:
According to the [official documentation](https://cobbler.readthedocs.io/en/latest/cobbler.html#cobbler-distro), the possible values for `breeds` are: `redhat|debian|suse` and it defaults to `redhat` but we explicitly set the default to `suse`, according to the method definition below.

Reply:
in that case:
```suggestion
  #   breed: The type of distribution.  This can be one of the following: redhat, debian, suse. Defaults to suse.
```